Bio/SeqIO/_index.py

```diff
@@ -140,6 +142,7 @@ def __iter__(self):
         SeqIO.SffIO._check_eof(handle, index_offset, index_length)
 
     def get(self, offset):
+        """Return the Sequence records at the given offset."""
```

Comment:
Please avoid the term "Sequence record" in favour of "SeqRecord", also this should be singular here not plural, how about:

```
    """Return the SeqRecord starting at the given offset."""
```

Reply:
(Same applies to the other ``get`` methods in this file)